### ISSUE TITLE
Clone failed on Windows

### ISSUE THREAD

**Issue by btconfig** [t-gao/mobilemouse#1]
error: unable to create file client/android/zxing-android-embedded/build/tmp/expandedArchives/internal_impl-23.1.0.jar_2267ci3lq90culadyhx28yfue/android/support/v4/view/accessibility/AccessibilityManagerCompatIcs$AccessibilityStateChangeListenerBridge.class (Filename too long)
error: unable to create file client/android/zxing-android-embedded/build/tmp/expandedArchives/internal_impl-23.1.0.jar_2267ci3lq90culadyhx28yfue/android/support/v4/view/accessibility/AccessibilityNodeProviderCompatJellyBean$AccessibilityNodeInfoBridge.class (Filename too long)
error: unable to create file client/android/zxing-android-embedded/build/tmp/expandedArchives/internal_impl-23.1.0.jar_aj566rqdrutol6a7nji1pql5d/android/support/v4/view/accessibility/AccessibilityManagerCompatIcs$AccessibilityStateChangeListenerBridge.class (Filename too long)
error: unable to create file client/android/zxing-android-embedded/build/tmp/expandedArchives/internal_impl-23.1.0.jar_aj566rqdrutol6a7nji1pql5d/android/support/v4/view/accessibility/AccessibilityNodeProviderCompatJellyBean$AccessibilityNodeInfoBridge.class (Filename too long)

**********************************************************************************************************
The four files can be deleted
